backend/core/test/applications/applications.e2e-spec.ts

```diff
@@ -457,7 +457,7 @@ describe("Applications", () => {
     await supertest(app.getHttpServer())
       .get(`/applications/${createRes.body.id}`)
       .set(...setAuthorization(user1AccessToken))
-      .expect(404)
+      .expect(500)
```

Comment:
Just curious why this changed here

Reply:
This change was also done in core, and we should look into why this changed there. I'll create an issue off of this.